lib/webview-helpers.js

```diff
@@ -292,23 +298,13 @@ helpers.procFromWebview = async function procFromWebview (adb, webview) {
 /**
  * Retrieves webview names for getContexts
  *
- * @param {object} adb - an ADB instance
+ * @param {Array<WebviewsMapping>} webviewsMapping See note on getWebViewsMapping
  * @param {GetWebviewOpts} opts See note on getWebViewsMapping
- *
  * @return {Array.<string>} - a list of webview names
  */
-helpers.getWebviews = async function getWebviews (adb, {
-  androidDeviceSocket = null,
-  ensureWebviewsHavePages = true,
-  webviewDevtoolsPort = null,
-  enableWebviewDetailsCollection = null,
+helpers.getWebviews = function getWebviews (webviewsMapping, {
```

Comment:
I think we could rename this method to `parseWebviewNames`

Reply:
Addressed as https://github.com/appium/appium-android-driver/pull/699/commits/4d2c0672a0fd0ede2a8d603e17487a7f415f6bd9